File: android/app/src/main/java/com/tonhub/wallet/modules/wallet/OPCRequest.java
package com.tonhub.wallet.modules.wallet;

import java.util.concurrent.CompletableFuture;

public class OPCRequest {
    String cardId;
    String token;
    String walletId;
    String stableHardwareId;
    Boolean isTestnet;
    CompletableFuture<String> future;

    public OPCRequest(String cardId, String token, String walletId, String stableHardwareId, Boolean isTestnet, CompletableFuture<String> future) {
        this.cardId = cardId;
        this.token = token;
        this.walletId = walletId;
        this.stableHardwareId = stableHardwareId;
        this.isTestnet = isTestnet;
        this.future = future;
    }
}

File: android/app/src/main/java/com/tonhub/wallet/modules/wallet/OPCRequest.kt
package com.tonhub.wallet.modules.wallet

import java.util.concurrent.CompletableFuture

data class OPCRequest(
    val cardId: String,
    val token: String,
    val walletId: String,
    val stableHardwareId: String,
    val isTestnet: Boolean,
    val future: CompletableFuture<String>
)


